packages/amplication-client/src/Layout/InnerTabLink.scss

```diff
@@ -19,7 +19,7 @@
   }
 
   &:last-child {
-    margin-bottom: 0;
+    margin-bottom: 8px;
```

Comment:
please use var

Reply:
margin bottom should be 0 for last child. Margin should be applied to a wrapper of list item and not to inner parts.